Update Plant Mortality tech note (#4075)
ESCOMP/CTSM: doc/source/tech_note/Plant_Mortality/CLM50_Tech_Note_Plant_Mortality.rst

.. _Table Model default annual mortality rate:

.. table:: Model default annual mortality rate (*r_mort*) for woody PFTs

   +---------------------------------------------------+--------------------------------------+
   | Plant functional type                             | Mortality rate (yr :sup:`-1`)        | 
   +===================================================+======================================+
   | NET Temperate                                     | 0.0211945164991821                   |
   +---------------------------------------------------+--------------------------------------+
   | NET Boreal                                        | 0.0174                               |
   +---------------------------------------------------+--------------------------------------+
   | NDT Boreal                                        | 0.0198950093389492                   |
   +---------------------------------------------------+--------------------------------------+
   | BET Tropical                                      | 0.024                                |
   +---------------------------------------------------+--------------------------------------+

>    | BET temperate                                     | 0.0199981934178915                   |

Reviewer: ```suggestion
   | BET Temperate                                     | 0.0199981934178915                   |
```

   +---------------------------------------------------+--------------------------------------+

>    | BDT tropical                                      | 0.0200001818014196                   |

Reviewer: ```suggestion
   | BDT Tropical                                      | 0.0200001818014196                   |
```

   +---------------------------------------------------+--------------------------------------+

>    | BDT temperate                                     | 0.0210684434513937                   |

Reviewer: ```suggestion
   | BDT Temperate                                     | 0.0210684434513937                   |
```

   +---------------------------------------------------+--------------------------------------+

>    | BDT boreal                                        | 0.024                                |

Reviewer: ```suggestion
   | BDT Boreal                                        | 0.024                                |
```

   +---------------------------------------------------+--------------------------------------+
 

> PFT name abbreviations: NET = Needleleaf Evergreen Tree, NDT = Needleleaf Deciduous Tree, BET = Broadleaf Evergreen Tree, BDT = Broadleaf Deciduous Tree

Reviewer: ```suggestion
PFT name abbreviations: NET = Needleleaf Evergreen Tree, NDT = Needleleaf Deciduous Tree, BET = Broadleaf Evergreen Tree, BDT = Broadleaf Deciduous Tree.  Shrubs and grasses all still use the default 2% annual mortality rate.
```

This section does not describe mechanistic mortality processes represented in the Functionally Assembled Terrestrial Ecosystem Simulator (FATES), where mortality emerges from explicit demographic, physiological, and disturbance processes (see Chapter :numref:`rst_Dynamic Global Vegetation and FATES`). Readers interested in those formulations should refer to the `FATES documentation`_. Mortality associated with fire and land-use or harvest processes is treated separately in the Fire and Land Use Change sections (see Chapters :numref:`rst_Fire` and :numref:`rst_Transient Landcover Change`, respectively). Legacy dynamic vegetation configurations (CNDV) used related mortality formulations but are no longer actively supported; mechanistic dynamic vegetation and mortality processes in CTSM are now handled through FATES.

.. _FATES documentation: https://fates-users-guide.readthedocs.io/en/latest/index.html

Implementation
----------------------------------------------

> Vegetation carbon and nitrogen dynamics are implemented using a matrix-based formulation (:ref:`Lu et al. 2020<Luetal2020>`), in which the evolution of vegetation pools is governed by a combination of process-specific transfer and turnover operators. Within this framework, mortality is represented as one component of the overall vegetation transfer system, alongside phenological turnover and fire-related processes.

Reviewer: ```suggestion
Vegetation carbon and nitrogen dynamics are implemented using a matrix-based formulation (:ref:`Lu et al. 2020<Luetal2020>`, :ref: `Liao et al. 2023`<Liaoetal2023>), in which the evolution of vegetation pools is governed by a combination of process-specific transfer and turnover operators. Within this framework, mortality is represented as one component of the overall vegetation transfer system, alongside phenological turnover and fire-related processes.
```

> Vegetation carbon and nitrogen dynamics are implemented using a matrix-based formulation (:ref:`Lu et al. 2020<Luetal2020>, Liao et al. 2023<Liaoetal2023>`), in which the evolution of vegetation pools is governed by a combination of process-specific transfer and turnover operators. Within this framework, mortality is represented as one component of the overall vegetation transfer system, alongside phenological turnover and fire-related processes.

Reviewer: ```suggestion
Vegetation carbon and nitrogen dynamics are implemented using a matrix-based formulation (:ref:`Lu et al. 2020<Luetal2020>`; :ref:`Liao et al. 2023<Liaoetal2023>`), in which the evolution of vegetation pools is governed by a combination of process-specific transfer and turnover operators. Within this framework, mortality is represented as one component of the overall vegetation transfer system, alongside phenological turnover and fire-related processes.
```

> Vegetation carbon and nitrogen dynamics are implemented using a matrix-based formulation (:ref:`Lu et al. 2020<Luetal2020>`, :ref:`Liao et al. 2023`<Liaoetal2023>), in which the evolution of vegetation pools is governed by a combination of process-specific transfer and turnover operators. Within this framework, mortality is represented as one component of the overall vegetation transfer system, alongside phenological turnover and fire-related processes.

Reviewer: ```suggestion
Vegetation carbon and nitrogen dynamics are implemented using a matrix-based formulation (:ref:`Lu et al. 2020<Luetal2020>`, :ref:`Liao et al. 2023<Liaoetal2023>`), in which the evolution of vegetation pools is governed by a combination of process-specific transfer and turnover operators. Within this framework, mortality is represented as one component of the overall vegetation transfer system, alongside phenological turnover and fire-related processes.
```

> Vegetation carbon and nitrogen dynamics are implemented using a matrix-based formulation (:ref:`Lu et al. 2020<Luetal2020>`, :ref:`Liao et al. 2023<Liaoetal2023>`), in which the evolution of vegetation pools is governed by a combination of process-specific transfer and turnover operators. Within this framework, mortality is represented as one component of the overall vegetation transfer system, alongside phenological turnover and fire-related processes.

Reviewer: ```suggestion
Vegetation carbon and nitrogen dynamics are implemented using a matrix-based formulation (:ref:`Lu et al. 2020<Luetal2020>, Liao et al. 2023<Liaoetal2023>`), in which the evolution of vegetation pools is governed by a combination of process-specific transfer and turnover operators. Within this framework, mortality is represented as one component of the overall vegetation transfer system, alongside phenological turnover and fire-related processes.
```

> Vegetation carbon and nitrogen dynamics are implemented using a matrix-based formulation (:ref:`Lu et al. 2020<Luetal2020>`, :ref: `Liao et al. 2023`<Liaoetal2023>), in which the evolution of vegetation pools is governed by a combination of process-specific transfer and turnover operators. Within this framework, mortality is represented as one component of the overall vegetation transfer system, alongside phenological turnover and fire-related processes.

Reviewer: ```suggestion
Vegetation carbon and nitrogen dynamics are implemented using a matrix-based formulation (:ref:`Lu et al. 2020<Luetal2020>`, :ref:`Liao et al. 2023`<Liaoetal2023>), in which the evolution of vegetation pools is governed by a combination of process-specific transfer and turnover operators. Within this framework, mortality is represented as one component of the overall vegetation transfer system, alongside phenological turnover and fire-related processes.
```

- Leaf and fine-root pools are transferred to litter pools (labile, cellulose, and lignin components)

- Stem and coarse-root pools are transferred to coarse woody debris pools

- Storage and transfer pools are assumed to represent labile material and are therefore transferred to litter pools

This formulation provides a simplified, bulk representation of mortality consistent with the structure of non-demographic CLM configurations. A conceptual diagram of vegetation fluxes and pools (:numref:`Figure Vegetation fluxes and pools`) can be found in Chapter :numref:`rst_CN Pools`.

Mortality fluxes leaving vegetation pools
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Whole-plant mortality is represented as a first-order process with a prescribed mortality rate. For an annual mortality fraction (:math:`a_m`, yr\ :sup:`-1`), the corresponding rate per unit time (second) :math:`m` is given by:

.. math::
   :label: 33.1)

   m = \frac{a_m}{365 \times 86400}

Mortality fluxes from vegetation carbon pools are computed as:

.. math::
   :label: 33.2)

   CF_{i,mort} =CS_{i} m

and similarly for nitrogen:

.. math::
   :label: 33.3)

   NF_{i,mort} =NS_{i} m

> where :math:`CF_{i}` is carbon flux, :math:`CS_{i}` is carbon storage, :math:`NF_{i}` is nitrogen flux, :math:`NS_{i}` is nitrogen storage, for each vegetation pool :math:`i`, respectively.

Reviewer: ```suggestion
where :math:`CF_{i}` is carbon flux, :math:`CS_{i}` is carbon state variable (or pool), :math:`NF_{i}` is nitrogen flux, :math:`NS_{i}` is nitrogen state, for each vegetation pool :math:`i`, respectively.
```
